eng/build.sh

```diff
@@ -14,20 +14,38 @@ scriptroot="$( cd -P "$( dirname "$source" )" && pwd )"
 
 usage()
 {
-  echo "Default if no actions are passed in: --restore --build"
+  echo "Common settings:"
+  echo "  --framework                The target group assemblies are built for (short: -f)"
+  echo "  --configuration <value>    Build configuration: 'Debug' or 'Release' (short: -c)"
+  echo "  --verbosity <value>        Msbuild verbosity: q[uiet], m[inimal], n[ormal], d[etailed], and diag[nostic] (short: -v)"
+  echo "  --binaryLog                Create MSBuild binary log (short: -bl)"
+  echo "  --help                     Print help and exit (short: -h)"
   echo ""
-  echo "CoreFx specific actions:"
-  echo "  --buildtests             Build test projects in the solution"
-  echo "  --clean                  Clean the solution"
+
+  echo "Actions (defaults to --restore --build):"
+  echo "  --restore                  Restore dependencies (short: -r)"
+  echo "  --build                    Build solution (short: -b)"
+  echo "  --buildtests               Build test projects in the solution"
+  echo "  --rebuild                  Rebuild solution"
+  echo "  --test                     Run all unit tests in the solution (short: -t)"
+  echo "  --integrationTest          Run all integration tests in the solution"
```

Comment:
Does corefx has any integration test? Skip this option?

Reply:
Not yet but I want to keep the option to add some. I think it doesn't harm to keep the action as the overall help message is already pretty compact now.

Reply:
So once we have them, we can add the option back.

(I doubt that integration tests make sense as an concept for corefx.)